geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/accumulo/commands/AddAttributeIndexCommand.scala

```diff
@@ -0,0 +1,71 @@
+/***********************************************************************
+* Copyright (c) 2013-2016 Commonwealth Computer Research, Inc.
+* All rights reserved. This program and the accompanying materials
+* are made available under the terms of the Apache License, Version 2.0
+* which accompanies this distribution and is available at
+* http://www.opensource.org/licenses/apache2.0.php.
+*************************************************************************/
+
+package org.locationtech.geomesa.tools.accumulo.commands
+
+import com.beust.jcommander.{JCommander, Parameter, Parameters}
+import com.typesafe.scalalogging.LazyLogging
+import org.apache.hadoop.util.ToolRunner
+import org.locationtech.geomesa.jobs.GeoMesaArgs
+import org.locationtech.geomesa.jobs.index.AttributeIndexJob
+import org.locationtech.geomesa.tools.accumulo.GeoMesaConnectionParams
+import org.locationtech.geomesa.tools.accumulo.commands.AddAttributeIndexCommand.AddIndexParameters
+import org.locationtech.geomesa.tools.common.{AttributesParam, FeatureTypeNameParam}
+
+import scala.util.control.NonFatal
+
+class AddAttributeIndexCommand(parent: JCommander) extends CommandWithCatalog(parent) with LazyLogging {
+  override val command = "add-attribute-index"
+  override val params = new AddIndexParameters
+
+  override def execute() = {
+    try {
+      val attributeIndexJobParams = Map(
+        GeoMesaArgs.InputUser -> params.user,
+        GeoMesaArgs.InputPassword -> params.password,
+        GeoMesaArgs.InputInstanceId -> params.instance,
+        GeoMesaArgs.InputZookeepers -> params.zookeepers,
+        GeoMesaArgs.InputTableName -> params.catalog,
+        GeoMesaArgs.InputFeatureName -> params.featureName,
+        AttributeIndexJob.IndexAttributes -> params.attributes,
+        AttributeIndexJob.IndexCoverage -> params.coverage
+      ).filter(_._2 != null).flatMap(e => List(e._1, e._2)).toArray
+
+      logger.info(s"Running map reduce index job for attributes: ${params.attributes} with coverage: ${params.coverage}...")
+
+      val result = ToolRunner.run(new AttributeIndexJob(), attributeIndexJobParams)
+
+      if (result == 0) {
+        logger.info("Add attribute index command finished successfully.")
+      } else {
+        logger.error("Error encountered running attribute index command. Check hadoop's job history logs for more information.")
+      }
+
+    } catch {
+      case npe: NullPointerException =>
+        logger.error(s"Error: feature '${params.featureName}' not found. Check arguments...", npe)
+      case e: Exception =>
+        logger.error(s"Error describing feature '${params.featureName}': " + e.getMessage, e)
+      case NonFatal(e) =>
+        logger.warn(s"Non fatal error encountered describing feature '${params.featureName}': ", e)
```

Comment:
this should be a logger.error - I would combine the case NonFatal and the case e: Exception and change the message to something more appropriate like the one on line 46.